src/Database/LMDB/Simple/TransactionHandle.hs

```diff
@@ -36,12 +36,12 @@ module Database.LMDB.Simple.TransactionHandle (
     TransactionHandle
   , abort
   , commit
-  , new
+  , _new
```

Comment:
With the underscores I wanted to reflect that we shouldn't be using these functions because they can be used with read-write transactions. I did still want to keep them exposed though

Reply:
Contrary to what I said, I've removed the definitions with underscores from the export list.

I did keep the underscores, because they prevent "Defined but not used" warnings (`-Wunused-top-binds`).